tests/gold_tests/h2/h2enable.test.py

```diff
@@ -69,29 +69,29 @@
 tr.Processes.Default.StartBefore(Test.Processes.ts)
 tr.StillRunningAfter = server
 tr.StillRunningAfter = ts
-tr.Processes.Default.TimeOut = 5
```

Comment:
Why did we need to increase the timeouts?  Was this a latent issue?

Reply:
Even without this change, the test doesn't complete within 5 seconds on my laptop. I'd say it's too strict.